src/tags/object/AudioNext/model.js

```diff
@@ -290,9 +290,9 @@ export const AudioModel = types.compose(
       },
 
       handleSeek() {
-        if (self._ws) {
-          self.triggerSyncSeek(self._ws.getCurrentTime());
-        }
+        if (!self._ws || self.syncedObject?.type === "paragraphs") return;
```

Comment:
need a tweak here to check for the FF - will need to import the FF higher up in the file too
```suggestion
        if (!self._ws || (isFF(FF_DEV_2461) && self.syncedObject?.type === "paragraphs")) return;
```